examples/chat_with_ai/main.py

```diff
@@ -5,10 +5,13 @@
 from langchain.chat_models import ChatOpenAI
 
 from nicegui import Client, ui
+from log_callback_handler import NiceGuiLogElementCallbackHandler
 
-OPENAI_API_KEY = 'not-set'  # TODO: set your OpenAI API key here
+OPENAI_API_KEY = "not-set"  # TODO: set your OpenAI API key here
```

Comment:
Please keep using single quotes in this example. The whole codebase is using it like this and we like consistency.